### ISSUE TITLE
FB11893679: SwiftUI Settings Scene Accessibility is broken in macOS 13.2. Toolbar Items are not Accessible anymore

### ISSUE THREAD

**Issue by godbout** [feedback-assistant/reports#374]
- **Date:** 2022-12-23
- **Resolution:** Open
- **Area:** SwiftUI Framework
- **OS:** macOS 13.2 beta 2
- **Type:** Incorrect/Unexpected Behavior

## Description

the Toolbar Items used to be accessible through the AX APIs. they are not anymore. this is easily seen by using the Accessibility Inspector: the Toolbar has now no children, where until 13.1 it would have the Items as children, that would therefore be reachable.

## Files

https://user-images.githubusercontent.com/121373/209311415-3956961d-654b-4893-a2ab-c979e78be145.mp4

https://user-images.githubusercontent.com/121373/209311468-d4fdb115-6118-42ea-9c60-b6ee675c1899.mp4

**Comment by godbout** [comment]
seems fixed in 13.3 beta 1. will close when fixed in prod.

**Comment by godbout** [comment]
Please verify this issue with macOS 13.3 Beta and update your bug report with your results by logging into https://feedbackassistant.apple.com/ or by using the Feedback Assistant app.

macOS 13.3 Beta (22E5219e)
https://developer.apple.com/download/
Posted Date: February 16, 2023

If the issue persists, please attach a new sysdiagnose captured in the latest build and attach it to the bug report. Thank you.

macOS sysdiagnose Instructions: 
https://developer.apple.com/services-account/download?path=/OS_X/OS_X_Logs/sysdiagnose_Logging_Instructions.pdf

___________________________________
For a complete list of logging instructions visit:
https://developer.apple.com/bug-reporting/profiles-and-logs/

**Comment by godbout** [comment]
Thank you for letting us know that your issue has been resolved. You can close this Feedback by selecting "Close Feedback" via the Actions button found above. As you’ve indicated this issue is resolved, this Feedback will no longer be monitored and incoming messages will not be reviewed. Should you find that the issue is still present, please file a new Feedback report.

We appreciate your feedback.
